trinity/protocol/common/trackers.py

```diff
@@ -76,39 +80,43 @@ def get_stats(self) -> str:
         """
         if not self.total_msgs:
             return 'None'
-        avg_rtt = self.total_response_time / self.total_msgs
-        if not self.total_response_time:
-            items_per_second = 0.0
-        else:
-            items_per_second = self.total_items / self.total_response_time
+
+        try:
+            rt99 = self.round_trip_99th.value
+        except ValueError:
+            rt99 = 0
```

Comment:
I was about to suggest this here as well.

```python
rt99 = 0 if not self.round_trip_99th.window else self.round_trip_99th.value 
```

But...it just occured to me that in this case the `try/except` may actually be faster than doing the `if/else` at every check. That's because the exception only happens at an interval of `window_size`. In that case :+1: